src/ATen/function_wrapper.py

```diff
@@ -8,12 +8,12 @@
     string_type = basestring
 
 # temporary things we cannot handle
-EXCLUDE_PATTERN = "bernoulli.*|normal.*|exponential.*|random.*|arange.*"
+EXCLUDE_PATTERN = "bernoulli.*"
```

Comment:
Does `bernoulli` still not work? I put in some effort to make it bindable in ATen.

Reply:
I wasn't able to get it to work. `BERNOULLI_TENSOR`, `bernoulli_FloatTensor`, and `bernoulli_DoubleTensor` aren't defined in ATen.

Reply:
Ugh I forgot about that part. I want to get this to work so I will look at how to fix this too.